parser/parser.go

```diff
@@ -26,16 +26,36 @@ type visitor struct {
 	name string
 }
 
-func (p *Parser) needType(comments string) (skip, explicit bool) {
-	for _, v := range strings.Split(comments, "\n") {
-		if strings.HasPrefix(v, structSkipComment) {
+func (p *Parser) needType(comments *ast.CommentGroup) (skip, explicit bool) {
+	if comments == nil {
+		return
+	}
+
+	for _, v := range comments.List {
+		comment := v.Text
+
+		if len(comment) > 2 {
+			switch comment[1] {
+			case '/':
+				// -style comment (no newline at the end)
+				comment = comment[2:]
+			case '*':
+				/*-style comment */
+				comment = comment[2 : len(comment)-2]
+			}
+		}
+
+		comment = strings.TrimSpace(comment)
+
+		if strings.HasPrefix(comment, structSkipComment) {
```

Comment:
@GoWebProd If I understand the code correctly, we no longer check each line of a multi-line comment here, only the first line of such comment, is that intended?

Reply:
I mean `/*...*/` style comment

Reply:
Yes, I never use it in go. I'm add support of multiline comments in master. Thanks.